Examples/Rule Sets/Questionnaires/enableWhen.fsh

```diff
@@ -0,0 +1,59 @@
+// this question will only be shown when another question has been answered as 'true'
```

Comment:
This description doesn't match the name of the RuleSet (`enableWhenCode`).  I think it might be left over from your previous RuleSet.